samples/README.MD

```diff
@@ -0,0 +1,44 @@
+# Samples, Snippets, and How-To Guides
+
+Developers like to learn by looking at code, and so the Azure SDK comes with a myriad of code samples in the form of short code snippets, sample applications, and how-to guides. This document describes where to find all these resources.
+
+# Structure of the Repository
+The Azure SDK repository is organized in the following folder structure, with the main sample locations highlighted using **bold** font.
+
+`/samples` - this folder<br>
```

Comment:
I'm assuming this should be bold too then

Reply:
not now; we don't [yet] have top level samples in this folder.